ccloud/docs/ccloud-demos-overview.rst

```diff
@@ -118,6 +118,10 @@ You can choose between two examples:
 
 .. figure:: ../../kubernetes/replicator-gke-cc/docs/images/operator-demo-phase-2.png
 
+Kafka DevOps
+------------
```

Comment:
Awesome.  Could you also please add a diagram?

Reply:
Diagram added and docs build tested locally against this branch